scripts/zkai_attention_kv_two_head_longseq_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,556 @@
+    spec.loader.exec_module(module)
+    return module
+
+
+SOURCE_INPUT_MODULE = load_script_module(
+    SOURCE_INPUT_SCRIPT, "zkai_attention_kv_stwo_native_two_head_longseq_bounded_softmax_table_proof_input"
+)
+
+
+def read_bounded_bytes(path: pathlib.Path, max_bytes: int, label: str) -> bytes:
+    if not path.is_file():
+        raise AttentionKvTwoHeadLongseqFusedSoftmaxTableGateError(f"missing {label}: {path}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)